js/modal1.js

```diff
@@ -0,0 +1,7 @@
+/*Ronjas modal script*/ 
+console.log("hello from modal 1"); //To see if it works
+
+export const handleBookButtonClick = ()=>{ ('#bookingModal');
```

Comment:
arrow function syntax should be: export const handleSomething = (prameter) =>{ 
code block;
}

I believe you intended to refer to the element by its ID. I recommend moving the ('#bookingModal') selector out of the block code entirely. Instead, declare it above as a constant.
Const modal/something = document.querySelector('#bookingModal');

Reply:
This one we fixed yesterday! all is good in the hood. :)